cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/service/http/BasicSystemHttpServiceContext.java

```diff
@@ -137,4 +148,12 @@ public void run(TxRunnable runnable) throws TransactionException {
       throw new IllegalArgumentException(String.format("Namespace '%s' does not exist", namespace), nfe);
     }
   }
+
+  @Override
+  public byte[] runTask(RunnableTaskRequest runnableTaskRequest) throws Exception {
+    String systemAppClassName = SystemAppTask.class.getCanonicalName();
```

Comment:
Why need to use `getCanonicalName`? `getName` should be ok.

Reply:
Done